src/core/unittest/FrameTest.cpp

```diff
@@ -173,7 +173,7 @@ TEST_P(AckFrameTest, DecodeAckFrameFail) {
         Buffer[2] = 1; // ACK Delay
         Buffer[3] = 0; // ACK range count
         Buffer[4] = 5; // First ACK range
-        for (auto TestValue : {64, 255}) {
+        for (uint8_t TestValue = 64; TestValue <= 255; TestValue++) {
```

Comment:
Why?

Reply:
Because otherwise it's an int, and then the Windows compiler throws an error about "loss of precision"

Reply:
Isn't this going to infinite loop? Since 255 is true, then rollover will happen and the condition will still be true.